ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/Readme.md

```diff
@@ -46,7 +46,7 @@ mvn archetype:generate
 mvn archetype:generate     
     \-DarchetypeGroupId=org.ikasan     
     \-DarchetypeArtifactId=ikasan-standalone-db-jms-im-maven-plugin 
-    \-DarchetypeVersion=2.1.0    
+    \-DarchetypeVersion=2.2.0    
```

Comment:
the 2.2.0 has not been released, so the instructions will fails if somebody tries to run the,.

Reply:
I didnt want to use 2.2.0-SNAPSHOT as we will forgot it on release. Need to look to see if we can get this automatic translating properly across the board. Hence I have set it to 2.2.0.